src/resources/views/quiz.blade.php

```diff
@@ -12,7 +12,7 @@
             <img src="/images/{{$question->image}}" alt="">
```

Comment:
altタグは記載しないのであれば削除してください
altタグは検索エンジン内の内部評価や視覚障害のある方のために音声読み上げ等で利用されるため画像には指定してあったほうが良いです
```suggestion
            <img src="/images/{{$question->image}}" alt="何かしらの説明が入っていると尚良いです">
```